src/puyapy_mocks/_reference.py

```diff
@@ -0,0 +1,75 @@
+from __future__ import annotations
+
+from puyapy_mocks import Bytes, UInt64
+from puyapy_mocks._ctx_state import active_ctx
+
+
+class Account:
+    def __init__(self, address: str):
+        self.address = address
+
+    def __eq__(self, other: Account | str) -> bool:  # type: ignore[override]
+        return True
+
+    def __ne__(self, other: Account | str) -> bool:  # type: ignore[override]
+        return True
+
+    def __bool__(self) -> bool:
+        return True
+
+    @property
+    def balance(self) -> UInt64:
+        return UInt64(active_ctx().balances[self.address]["0"])
```

Comment:
0 always assumes algo balance?

Reply:
Most of these implementations were to just make it work, so will need to be properly implemented when we build that outcome for real.